osu.Game/Skinning/SkinImporter.cs

```diff
@@ -179,8 +179,14 @@ void writeNewSkinIni()
 
         private Skin createInstance(SkinInfo item) => item.CreateInstance(skinResources);
 
-        public void Save(Skin skin)
+        /// <summary>
+        /// Save a skin. Updates any drawable layouts that are out of date.
```

Comment:
This xmldoc confuses me. To me it seems to imply that something about drawables is being done here but all that this method does is serialise JSON files?